decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/highlighted_processes/single_process.erb

```diff
@@ -14,7 +14,7 @@
                 <%= link_to decidim_participatory_processes.participatory_process_path(process), class: "card__link" do %>
                   <h2 class="card__title" id="<%= title_id %>"><%= decidim_html_escape(translated_attribute(process.title)) %></h2>
                 <% end %>
-                <%= decidim_sanitize_editor translated_attribute(process.short_description) %>
+                <%= decidim_sanitize_editor_admin translated_attribute(process.short_description) %>
```

Comment:
I'm not quite sure if we should allow embeds in the cards. It makes the card really large and not very usable.

Do you have any concrete example where it would be required?

Reply:
I checked it and I agree it almost breaks the layout.
I don't have any requirement to change this. It is reverted in the update.